reports/src/main/java/com/ivy/reports/ReportScreenState.kt

```diff
@@ -1,31 +1,36 @@
 package com.ivy.reports
 
-import com.ivy.data.AccountOld
-import com.ivy.data.CategoryOld
+import androidx.compose.runtime.Stable
+import com.ivy.data.account.Account
+import com.ivy.data.category.Category
 import com.ivy.data.transaction.TransactionOld
+import com.ivy.data.transaction.TransactionsList
 import java.util.*
 
+@Stable
 data class ReportScreenState(
     val baseCurrency: String = "",
     val balance: Double = 0.0,
+
     val income: Double = 0.0,
+    val incomeTransactionsCount: Int = 0,
+
     val expenses: Double = 0.0,
-    val upcomingIncome: Double = 0.0,
-    val upcomingExpenses: Double = 0.0,
-    val overdueIncome: Double = 0.0,
-    val overdueExpenses: Double = 0.0,
-    val history: List<Any> = emptyList(),
-    val upcomingTransactions: List<TransactionOld> = emptyList(),
-    val overdueTransactions: List<TransactionOld> = emptyList(),
-    val categories: List<CategoryOld> = emptyList(),
-    val accounts: List<AccountOld> = emptyList(),
-    val upcomingExpanded: Boolean = false,
-    val overdueExpanded: Boolean = false,
+    val expenseTransactionsCount: Int = 0,
+
+    val accounts: List<Account> = emptyList(),
+    val categories: List<Category> = emptyList(),
+
+    val showTransfersAsIncExpCheckbox: Boolean = false,
+    val treatTransfersAsIncExp: Boolean = false,
+
     val filter: ReportFilter? = null,
+    val transactionsWithDateDividers: TransactionsList = emptyTransactionList(),
```

Comment:
Simplify to `trnsList` - no one care if it's with or w/o date dividers and they can find this info in `TransactionsList` type.